scripts/generated/tooltips.ts

```diff
@@ -35,7 +35,8 @@ export const TOOLTIP_STRING_KEYS = {
 
     // GENERAL
 
-    DYDX_ADDRESS_BODY: 'TOOLTIPS.GENERAL.DYDX_ADDRESS.BODY',
```

Comment:
technically we need to ensure mobile isn't using this since this is a breaking change

Reply:
ooo maybe i should just add a new one?

Reply:
https://github.com/search?q=org%3Adydxprotocol+DYDX_ADDRESS_BODY&type=code doesnt looook like theyre using it but i asked in the channel